### ISSUE TITLE
3D Kidneys and Kidney Tumor Semantic Segmentation using Boundary-Aware Networks

### ISSUE THREAD

**Issue by zerebom** [zerebom/paper-books#9]
src:
https://arxiv.org/abs/1909.06684

腎臓のDice score:0.9742
腎臓がんのDice score:0.8103
全体:0.8923と高い精度を出している。

**Comment by zerebom** [comment]
## Abstract
Kits19のデータセットを使って、腎臓がんの自動識別をEncoder-Decoderモデルで行った論文。    


従来のEncoder-Decorder機構にプラスして、境界面を特に注目するBoundary-Streamをネットワークに追加した。  

## Introduction
### Architecture
2018年の腎臓がんの死者は延べ175,000人もいる。  
現在の腎臓がんに対する手術は腎臓を全て除去するRN(Radical Nephrectomy)と、
癌になっている部分だけを取り除くPN(Partial Nephrectomy)がある。  
癌の早期発見をすることができれば、手術を軽いPNだけですませることができるようになる。  

高い専門技術が必要かつ、ミスが起こりやすい癌の識別という作業をNNによって代替できれば、
死者も医者の負担も減らすことができる。

## Methods
![image](https://user-images.githubusercontent.com/38466611/68072029-687dc100-fdc4-11e9-9dc9-ed470d58fce6.png)
### Architecture
3x176x176x176のCT画像を入力し、学習する。
最初は16のフィルターを持つ3x3x3の3dConvに渡す。

### Loss Functions
ResidualBlock

### Implementation Details and Dataset【重要】
#### Data processing
[-1024,1024]->を1000分の1にして[-1,1]にする。そこから外れたものは削る
specingは1x1x1mmにする。
cropは176x176x176にする。

#### Implementation details
Pytorch
0.8は cropping region was centered on the kidney tumor
0.1は任意のForeground
0.1はBackground

8x NVIDIA Tesla V100 G16


## Results
